active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,662 @@
+In a UWP app you must create a new 
+[CoreApplicationView](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationView),
+which will automatically create a new thread and a Window on it. 
+See the 'Create new Window in UWP application' example.
+
+> Implementation note: Attempting to activate a new window in a UWP app will use 
+[RoOriginateError](https://docs.microsoft.com/en-us/windows/win32/api/roerrorapi/nf-roerrorapi-rooriginateerror) 
+to show an explanatory message in the debugger.
+
+Creating a new Window in a Desktop app creates a new top level HWND.
+
+## Window.ExtendsContentIntoTitleBar
+
+Gets or sets a value that specifies whether the default window title bar should be removed, creating more space for content.
+
+Setting this property to true causes the built-in title bar to be removed. 
+
+> Spec note: this is analogous to 
+[AppWindowTitleBar.ExtendsContentIntoTitleBar](https://docs.microsoft.com/uwp/api/Windows.UI.WindowManagement.AppWindowTitleBar.ExtendsContentIntoTitleBar),
+and the implementation uses that property in UWP.
+
+Setting this property only effects rendering, it does not affect pointer (such as mouse) behavior. 
+For example, you can still use the mouse to click down in the area at the top of the window and 
+drag the window around the desktop. To change that behavior use the `SetTitleBar` API.
+
+Extending the content into the title bar does not impact the Window buttons 
+(Minimize, Maximize, and Close).
+
+See the `SetTitleBar` example below for an example of ExtendsContentIntoTitleBar
+
+## Window.SetTitleBar 
+_Behavior update to existing API when running as Desktop app_
+
+Makes a XAML element interact with the system as if it’s the title bar.
+
+For example, a user can move the window by dragging the XAML UIElement, 
+or a user can invoke the window context menu by right-clicking it. As a consequence the app no longer
+receives pointer input when the user interacts with the target UIElement or its children using touch, mouse, or pen. 
+The UIElement does still receive keyboard input.
+
+Only one UIElement can act as Title bar, so the last set wins. To use multiple objects, 
+you need to wrap them in a container element (for example a Grid).
+
+The rectangular area occupied by the element acts as the title bar for pointer purposes,
+even if the element is blocked by another element,
+even if the element is transparent.
+
+This method is typically used with the Window's `ExtendContentIntoTitleBar` property 
+set to true in order to hide the default system title bar.
+However, even when the default system title bar is not hidden, SetTitleBar can be used to make 
+additional regions in your app behave like the title bar.
+
+The following shows a Window with a no built-in title bar. 
+Instead it uses the whole window as its content area. To enable features such as window dragging, 
+it designates a TextBlock as the title bar.
```

Comment:
Do you want the example to use the TextBlock as the titlebar, or the <Grid> containing the TextBlock?  If so, the Grid should also use a background color of **Transparent** for hit testing.